requirements/base.txt

```diff
@@ -6,3 +6,4 @@ requests>=1.0.0
 tornado>=4.2.1
 # Required by Tornado to handle threads stuff.
 futures>=2.0
+jsonschema==2.6.0
```

Comment:
Since this is disabled by default I don't think jsonschema needs to be a required base dep

Reply:
Totally agree. We can come back to this question in future releases if this takes off but for the time being, I don't feel like we can accept this as a hard dependency for Salt.